src/lib.rs

```diff
@@ -2,20 +2,21 @@ mod altered_game;
 mod board;
 mod clock;
 mod coord;
-mod event;
 mod force;
-mod game;
 mod grid;
 mod piece;
 mod player;
 mod rules;
 mod scores;
 pub mod client;
+pub mod event;
```

Comment:
All modules in this crate are already public. Some directly, some (like `event` and `game`) — through a public use `module::*` (see below)

Reply:
Done